### PR TITLE
add pointers in cslam for intel performance (and update cime external?) for cam_cesm2_2_rel branch

### LINKED ISSUE #955: Update CSLAM Derecho performance issues in 2.2 release tag
### Issue Type

Other (please describe below)

### Issue Description

Derecho performance issues for CSLAM were resolved on the trunk https://github.com/ESCOMP/CAM/pull/845? but not on the 2.2 release tag. This is needed for running CSLAM with the 2.2 release tag.

There is also an issue with i/o via the mpich compiler, which will be resolved with a new 2.2 cime tag @fvitt. Ideally that tag wold be made first and would update the externals in this issue as well.

### Will this change answers?

I Don't Know

### Will you be implementing this yourself?

Any CAM SE can do this

### PR DESCRIPTION
**Pull request by adamrher**
Fixes #955 

For the new cesm2.2 tag, we will need to update the cime external as in https://github.com/ESMCI/cime/pull/4559 i/o performance issues with VR grids. Should we update the external in the cam tag?

### CODE REVIEW

**Comment by adamrher** [comment]
Updated cime exterrnal and ran a cslam test case to completion. The cime tag has a weird name ```maint-5.8_5.8.32``` but I think it's the correct tag. Provided it is, I think regressions testing can start.

File: Externals.cfg [hunk 1]
```diff
@@ -9,7 +9,8 @@ local_path = components/cice
 required = True
 
 [cime]
-tag = cime5.8.32.7
+#tag = cime5.8.32.7
+tag = maint-5.8_5.8.32
```
Comment (cacraigucar) [inline Externals.cfg:13]: I have requested a tag to be made for the head of this branch.  When it is made, we will use that tag.

File: src/dynamics/se/dycore/fvm_consistent_se_cslam.F90 [hunk 2]
```diff
@@ -152,22 +154,26 @@ subroutine run_consistent_se_cslam(elem,fvm,hybrid,dt_fvm,tl,nets,nete,hvcoord,&
...
              fvm(ie)%vertex_recons_weights,fvm(ie)%vtx_cart,&
              irecons_tracer_lev(k))
-        !call t_stopf('fvm:tracers_reconstruct')
-        !call t_startf('fvm:swept_flux')
-        call swept_flux(elem(ie),fvm(ie),k,ctracer,irecons_tracer_lev(k),gsweights,gspts)
-        !call t_stopf('fvm:swept_flux')
+         !call t_stopf('FVM:tracers_reconstruct')
+         !call t_startf('fvm:swept_flux')
+         call swept_flux(elem(ie),fvm(ie),k,ctracer,irecons_tracer_lev(k),gsweights,gspts)
+         !call t_stopf('fvm:swept_flux')
```
Comment (cacraigucar) [inline src/dynamics/se/dycore/fvm_consistent_se_cslam.F90:176]: I know you didn't make this change, but I'm wondering why these timers were commented out.  I would suggest it makes sense to uncomment them, so they are available if folks wanted to time these routines.

**Review by cacraigucar — CHANGES_REQUESTED** [review]
If I have access to your fork, and you are okay with it, I can put in the updated tag when it is made.  I will also need to make a ChangeLog for this commit, and can add it when it is ready.

File: Externals.cfg (hunk 1 above)
Comment (cacraigucar) [reply Externals.cfg:13]: The tag has been made and it is:  5.8.32.10

**Review by cacraigucar — COMMENTED** [review]
Still need to squash bugs, but the code looks fine

**Review by cacraigucar — APPROVED** [review]

**Comment by cacraigucar** [comment]
It was decided to not continue supporting the cesm2_2 branch, so this PR is being closed without being implemented.

**Pull request closed without merging**